sdk/src/wallet/account/operations/transaction/prepare_output.rs

```diff
@@ -110,6 +110,13 @@ where
         let min_storage_deposit_basic_output =
             MinimumStorageDepositBasicOutput::new(rent_structure, token_supply).finish()?;
 
+        let min_required_storage_deposit = if nft_id.is_some() {
+            let min_storage_deposit_nft_output = Output::Nft(first_output.as_nft().clone()).rent_cost(&rent_structure);
+            min_storage_deposit_basic_output.max(min_storage_deposit_nft_output)
```

Comment:
Not asking to remove the max, but isn't Nft assured to be > Basic?

Reply:
In this case true, only if the basic output has features or more unlock conditions it could be different, but currently this isn't checked at this place, will check tomorrow if this also needs to be considered at this place or not

Reply:
Updated it